include/bluetooth/bluetooth.h

```diff
@@ -277,6 +277,16 @@ enum {
 	 *  bt_conn_create_slave_le().
 	 */
 	BT_LE_ADV_OPT_DIR_MODE_LOW_DUTY = BIT(4),
+
+	/** Enable use of Private Resolvable Address in directed advertisements
+	 *  when CONFIG_BT_PRIVACY is not enabled.
+	 *  This is required if the remote device is privacy enabled and
+	 *  supports address resolution of directed advertisement.
+	 *  It is the responsibility of the application to check that the remote
+	 *  device supports address resolution of directed advertisements by
+	 *  reading it's Central Address Resolution characteristic.
```

Comment:
*its

Reply:
@joerchan this doesn't seem to be fixed yet?